Automattic/jetpack
# Stats Admin: real site locale for odyssey

Fixes https://github.com/Automattic/wp-calypso/pull/75190#discussion_r1159025078

## Proposed changes:

The PR changes the locale in `configData` passed to the Odyssey Apps (Stats and Widget) from only the language code e.g. `en` to full locale `en-us`.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

- Spin up a JN site
- Ensure Odyssey Stats doesn't break although translations / date formats might not work
- Test with https://github.com/Automattic/wp-calypso/pull/75363
- Ensure translations work well for `en-us`, `en-gb`, `es-ar`, `es`, `zh-tw` <- change language from `/wp-admin/options-general.php`
- Ensure date time format work well for `en-us`, `en-gb`, `es-ar`, `es`, `zh-tw`
